### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by aeivy** [tbodt/ish#587]
Hello,
To make it short and quick, I was trying to setup ssh on my iPad and once I had reached /etc/ssh/sshd_config, it throws out an error telling me that my permission is denied. I even tried /etc/motd and the same happened. What gives?

**Comment by tbodt** [comment]
Did you create a user account?

**Comment by aeivy** [comment]
I don’t think I have.
I do know it’s the `adduser` command but what parameters do I fill it with. I searched the wiki and didn’t find anything.

**Comment by tbodt** [comment]
Was asking because adding a user would make it less likely to work :)

Honestly I have no idea. If you're logged in as root you shouldn't get that error.

**Comment by aeivy** [comment]
Forgive me for being blunt as I’m new to the shell. After firing up the shell and typing in 
`whoami` it returns `root`. To note, in the shell and before typing anything Instead of having the $ sign, I have a #, which based on what I know, means that I’m a root user.

**Comment by tbodt** [comment]
Yes, that means you're root. I have no idea what's wrong, but can you run `ls -l /etc/ssh/sshd_config` and send me the output?

**Comment by aeivy** [comment]
Sure thing!
`-rw-r--r--    1 root     root          3177 Oct 14 14:48 /etc/ssh/sshd_config `

**Comment by tbodt** [comment]
I'm out of ideas on this one. Could you do this? https://github.com/tbodt/ish/wiki/Extracting-a-dump-of-your-files

**Comment by abrobin2802** [comment]
Same issue here

**Comment by Cookupfool** [comment]
Hey you ever figure out the problem?
